src/rpdk/core/contract/suite/handler_commons.py

```diff
@@ -154,8 +154,15 @@ def test_input_equals_output(resource_client, input_model, output_model):
     pruned_output_model = prune_properties_if_not_exist_in_path(
         output_model.copy(), pruned_input_model, resource_client.read_only_paths
     )
+
     pruned_output_model = prune_properties_if_not_exist_in_path(
         pruned_output_model, pruned_input_model, resource_client.create_only_paths
     )
```

Comment:
Nice, I think we can remove this `createOnlyProperty` pruning now
```suggestion
```
https://github.com/aws-cloudformation/cloudformation-cli/pull/531#discussion_r469462358